src/web/arr/trove/color.arr

```diff
@@ -52,6 +52,7 @@ gold = color(255, 215, 0, 1)
 yellow = color(255, 255, 0, 1)
 olive = color(128, 128, 0, 1)
 burlywood = color(222, 184, 135, 1)
+amaranth = color(229, 43, 80, 1)
```

Comment:
Just to give you a headsup, this also needs an update at the `pyret-docs` repo, specifically on this file: https://github.com/brownplt/pyret-docs/blob/master/src/trove/color.js.rkt

Reply:
Where did this color come from?  It's not an X11 color, or a DrRacket color, and I would like to have some sort of systematic reason for adding new colors...

Reply:
We mainly added this color because we wanted to test if we set up our gitHub branch and local pyret repl correctly. I guess after we confirmed that our commits actually result in changes to the local repl, we just moved on and left that in. 

The reason that we chose Amaranth is that we to quickly find a color that wasn’t already on the list of colors so we looked up the most underrated colors and picked one we liked from [here](https://www.mentalfloss.com/article/53121/11-colors-youve-probably-never-heard)

I’m perfectly okay with removing that color now if it’s unnecessary and goes against the current system of colors.